File: src/main/java/org/cobraparser/CobraParser.java
package org.cobraparser;

import java.util.logging.Level;
import java.util.logging.Logger;

public class CobraParser {

    private static Logger logger = Logger.getLogger(CobraParser.class.getName());
    private static boolean debug = false;

    public static boolean isDebugOn() {
        return CobraParser.debug;
    }

    public static void setDebug(boolean debug) {
        CobraParser.debug = debug;
        logger.log(Level.INFO, "Logging has been " + (debug ? "Enabled" : "Disabled"));
    }

}

File: src/main/kotlin/org/cobraparser/CobraParser.kt
package org.cobraparser

import java.util.logging.Level
import java.util.logging.Logger

object CobraParser {

    private val logger = Logger.getLogger(CobraParser::class.java.name)

    @JvmField
    var isDebugOn = false

    @JvmStatic
    fun setDebug(debug:Boolean) {
        CobraParser.isDebugOn = debug
        logger.log(Level.INFO, "Logging has been " + (if (debug) "Enabled" else "Disabled"))
    }

}